apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -150,6 +151,47 @@ Page {
                 }
                 onClicked: pageStack.push("PermissionPage.qml")
             }
+
+            BrowserListItem {
+                //% "Save destination"
+                label: qsTrId("settings_browser-la-save_destination")
+                iconSource: "image://theme/icon-m-download"
+                value: {
+                    if (WebEngineSettings.useDownloadDir) {
+                        //% "Download dir"
+                        return qsTrId("sailfish_browser-me-download_dir")
+                    } else {
+                        //% "Ask"
+                        return qsTrId("sailfish_browser-me-ask")
+                    }
+                }
+
+                secondaryText: {
+                    if (WebEngineSettings.useDownloadDir) {
+                        //% "Downloaded files will be saved to %1"
```

Comment:
Could be "Files will be downloaded to %1"

Reply:
Please ignore my earlier comment. Based on the design the string should be "Downloaded files will be saved to %1 folder".